nixos/doc/manual/release-notes/rl-2009.xml

```diff
@@ -277,6 +277,9 @@ environment.systemPackages = [
   <title>Other Notable Changes</title>
 
   <itemizedlist>
+   <listitem>
+     <para>SD and ISO images are now compressed by default using <literal>zstd</literal>.</para>
```

Comment:
This is false, only the SD image is now compressed by default using `zstd`.

The ISO image needs to be configured to be compressed to be compressed, and *then* it would now be `zstd`.